spec/features/saml/loa3_sso_spec.rb

```diff
@@ -34,14 +34,26 @@
   end
 
   context 'canceling verification', js: true do
+    it 'returns user to recovery code page if they sign up via loa3' do
+      user = create(:user, phone: '1 (111) 111-1111', recovery_code: nil)
+      sign_in_with_warden(user)
+      loa3_sp_session
+
+      visit verify_path
+
+      click_on t('links.cancel')
+      click_on t('idv.buttons.cancel')
+      expect(current_url).to match(/recovery_code/)
```

Comment:
That doesn't work in this case. Running the specs with javascript enabled produces `http://127.0.0.1:50114/` instead of `http://www.example.com/` as the url